src/rhsm/profile.py

```diff
@@ -75,8 +75,15 @@ def _uniquify(module_list):
         ret = {}
         for module in module_list:
             key = (module["name"], module["stream"], module["version"], module["context"], module["arch"])
-            ret[key] = module
-        return list(ret.values())
+            # Prefer duplicates that are Active.
+            # There are "enabled" duplicates from dnf that are marked as inactive.
+            if ret.get(key) != None:
+                if ret[key]["active"] != True:
+                    ret[key] = module
+            else:
+                ret[key] = module
+
+    return list(ret.values())
```

Comment:
Is the indentation of the return statement correct? I thought for python indentation is crucial...

Reply:
Yeah that's definitely not right! needs to be one indent in looks like